freeipa.spec.in

```diff
@@ -306,6 +306,7 @@ Requires: oddjob
 Requires: gssproxy >= 0.7.0-2
 # 1.15.2: FindByNameAndCertificate (https://pagure.io/SSSD/sssd/issue/3050)
 Requires: sssd-dbus >= 1.15.2
+Requires: python-augeas
```

Comment:
We want to migrate *all* config file handling to augeas eventually, now is as good time as any to add the dependency.